Fails to build on FreeBSD 13.0-RELEASE

Fails to build on FreeBSD 13.0-RELEASE:

```
FreeBSD% LANG=C cmake ..   
-- Configuring done
CMake Error: AUTOMOC for target QMidi: Could not find moc executable target Qt5::moc
CMake Generate step failed.  Build files cannot be regenerated correctly.
```

Unfortunately that's not it. Same error message. I can build other Qt projects just fine on this machine. This is what is puzzling me.

Another project:

```
FreeBSD% truss cmake .. 2>&1 | grep moc
access("/usr/local/lib/qt5/bin/moc",R_OK)        = 0 (0x0)
access("/usr/local/lib/qt5/bin/moc",R_OK)        = 0 (0x0)
fstatat(AT_FDCWD,"/usr/local/lib/qt5/bin/moc",{ mode=-rwxr-xr-x ,inode=699497,size=910800,blksize=131072 },0x0) = 0 (0x0)
```

This project:

```
FreeBSD% truss cmake .. 2>&1 | grep moc

CMake Error: AUTOMOC for target QMidi: Could not find moc executable target Qt5::mocwrite(2,"CMake Error: AUTOMOC for target "...,84) = 84 (0x54)
```

Maybe it's _something_ in the CMake files.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,7 @@ set(CMAKE_AUTORCC ON)
 set(CMAKE_AUTOMOC ON)
 
 find_package(QT NAMES Qt6 Qt5 COMPONENTS Core REQUIRED)
-#find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Core REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Core REQUIRED)
 
 # Common QMidi source files & library
 aux_source_directory("${PROJECT_SOURCE_DIR}/src" SOURCES)

